### DISCUSSION TITLE
iconified window on the left side

### DISCUSSION THREAD

**Discussion by zeroheure** [trunkmaster/nextspace#386] [Ideas]
Current screen are a lot wider than high. I think it would make sense to put iconified window on the left side instead of bottom (with an option of course).

**Comment by trunkmaster** [comment]
Technically it's not so hard, but from design point of view it can be a problem. 

At left side application menu resides by default. This arises some questions:
1. Were menu should appear by default (cover miniwindows, just at right from miniwindows)?
2. Multi-monitor configuration: usually monitors are placed side by  side. If someone wants to place window occupying both monitors it will overlap with miniwindows.
3. Dock can be dragged from right side to the left. And it makes sense if someone want to place Dock at left monitor in multi-monitor configuration (see case 2).

Anyway, these questions are worth to work in the future. Leave this issue without "Milestone" label.
